bin/buildBlueprints.js

```diff
@@ -12,37 +13,35 @@ var makeBuild = require('../lib/makeBuild').makeBuild;
 var configs = require('../lib/configs');
 var getWebpackEntryForTest = require('../lib/getWebpackEntryForTest');
 
+var PRODUCTION_ENV = 'production';
+var TARGETS = {
```

Comment:
🔕  I feel more a bit more strongly about this now. We have custom configs in reddit mobile and it would be nice to not have to hardcode `options.env === 'production'`